batchflow/models/tf/deep_galerkin.py

```diff
@@ -433,9 +528,95 @@ def _callable(shifted_time):
 
         return _callable
 
+
     def predict(self, fetches=None, feed_dict=None, **kwargs):
         """ Get network-approximation of PDE-solution on a set of points. Overloads :meth:`.TFModel.predict` :
         `solution`-tensor is now considered to be the main model-output.
         """
         fetches = 'solution' if fetches is None else fetches
         return super().predict(fetches, feed_dict, **kwargs)
+
+
+
+class DGSolver():
+    """ Wrapper around `DeepGalerkin` to surpass BatchFlow's syntax sugar.
+
+    Parameters
+    ----------
+    model_config : dict
+        Configuration of model. Supports all of the options from `DeepGalerkin`.
+    """
+
+    def __init__(self, model_config, dg_class=DeepGalerkin):
```

Comment:
```suggestion
    def __init__(self, model_class=DeepGalerkin, config=None):
```
To comply with Pipeline API